.github/workflows/test-all-metadata.yml

```diff
@@ -34,7 +34,7 @@ jobs:
   test-all-metadata:
     name: "🧪 ${{ matrix.coordinates }} (GraalVM ${{ matrix.versions.graalvm }} ${{ matrix.versions.java }} @ ${{ matrix.os }})"
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 20
+    timeout-minutes: 80
```

Comment:
Why do we need to increase timeout by 4x? How long does it take to build https://github.com/oracle/graalvm-reachability-metadata/issues/163#issuecomment-1418723038 on GitHub Actions? I'd prefer if we could selectively bump the timeout for metadata that requires it.

Reply:
- Completing nativeTest for `org.apache.curator:curator-recipes:5.4.0` took 62 minutes on my device. I've created the smallest possible subset of unit tests for this dependency on my local device, but the full test still takes so long.

>  I'd prefer if we could selectively bump the timeout for metadata that requires it.

-  I don't quite understand, how to do this?

Reply:
> Completing nativeTest for `org.apache.curator:curator-recipes:5.4.0` took 62 minutes on my device.

That is quite a lot. Were these 64min dominated by Native Image builds or by actually running the tests?

> I don't quite understand, how to do this?

Right now, we don't have a mechanism for this, but maybe we can come up with something that allows overriding the timeout if necessary. We could implement that in [`org.graalvm.internal.tck-harness.gradle`](https://github.com/linghengqian/graalvm-reachability-metadata/blob/985f371e5dc9fc5a460619359166ed07ebe5d38d/tests/tck-build-logic/src/main/groovy/org.graalvm.internal.tck-harness.gradle) somehow, and then use:

```yml
timeout-minutes: ${{ matrix.timeout-minutes }}
```

Reply:
Do we need to run the unit tests of `curator-recipes` on the JVM or can we skip this step?

Reply:
- I don't think it's necessary to run unit tests on the JVM, it's definitely something to consider. But, what I see in https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html#testing-support says
```
Currently, this feature requires the execution of the tests in the classic "JVM" mode prior to the execution of tests in native mode. 
```
- I honestly don't see an option to disable nativeTest from executing unit tests on the JVM, is there something I'm overlooking here?

Reply:
>I honestly don't see an option to disable nativeTest task from executing unit tests on the JVM, is there something I'm overlooking here?

You are right, I'm afraid tests currently need to run on the JVM so that we can reliably detect all test ids.

Reply:
- So should I close this PR now?

Reply:
You can unless you want to look into making timeouts adjustable per metadata entry, which I believe we will need to support `curator-recipes`.